src/Swarm/Game/Failure.hs

```diff
@@ -12,12 +12,13 @@ import Data.Yaml (ParseException)
 
 data SystemFailure
   = AssetNotLoaded Asset FilePath LoadingFailure
+  deriving (Show)
```

Comment:
Took me a while to find how this is pretty printed, so I added `Show` instance while debugging.

Should the printing function be here or at least a link to it?

Reply:
Adding more documentation is always a good thing!

Reply:
Done!